cypress/e2e/brewery/ScenarioParameters-additional-advanced.cy.js

```diff
@@ -88,6 +94,11 @@ describe('Additional advanced scenario parameters tests', () => {
       .type('{moveToStart}' + '29/08/1997')
       .should('value', '29/08/1997');
 
+    BreweryParameters.selectScenarioToCompareOption(VALUES_TO_UPDATE.scenarioToCompare).should(
```

Comment:
I'm not sure about this assertion because the used function `selectScenarioToCompareOption` doesn't return an element to assert for :thinking: